src/services/codefixes/helpers.ts

```diff
@@ -63,7 +63,6 @@ namespace ts.codefix {
         const visibilityModifier = createVisibilityModifier(getEffectiveModifierFlags(declaration));
         const modifiers = visibilityModifier ? factory.createNodeArray([visibilityModifier]) : undefined;
         const type = checker.getWidenedType(checker.getTypeOfSymbolAtLocation(symbol, enclosingDeclaration));
-        const optional = !!(symbol.flags & SymbolFlags.Optional);

```

Comment:
I think we still need this, since if the new member is a property declaration, we need to preserve optionality.
I think we can either check if we should preserve optionality here, or have this function take a `preserveOptional` parameter or something like that.

Reply:
Oke, thanks